src/main/java/core/basesyntax/strategy/handler/impl/AddToStorage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.strategy.handler.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class AddToStorage {
+    private static final int DEFAULT_VALUE = 0;
+
+    public void add(FruitTransaction fruitTransaction) {
+        int currentQuantity = Storage.storage
+                .getOrDefault(fruitTransaction.getFruit(), DEFAULT_VALUE);
+        Storage.storage.put(fruitTransaction.getFruit(),
+                currentQuantity + fruitTransaction.getQuantity());
+    }
+}
```

Comment:
In my opinion, it's better to rename this class to StorageDao and don't forget about SOLID principles